src/EnergyPlus/DXCoils.cc

```diff
@@ -14979,7 +14979,7 @@ void CalcTwoSpeedDXCoilStandardRating(EnergyPlusData &state, int const DXCoilNum
     bool ErrorsFound(false);
     Real64 FanHeatCorrection;
     Real64 FanPowerCorrection;
-    Real64 FanPowerPerEvapAirFlowRate;
+    Real64 FanPowerPerEvapAirFlowRate = 0.0;
```

Comment:
This is an unrelated unit test repair.

Reply:
Interesting.  OK well no problem, obviously.